x/warp/contract.go

```diff
@@ -191,18 +184,18 @@ func getVerifiedWarpMessage(accessibleState contract.AccessibleState, caller com
 	return handleWarpMessage(accessibleState, input, suppliedGas, addressedPayloadHandler{})
 }
 
-// UnpackSendWarpMessageInput attempts to unpack [input] as SendWarpMessageInput
+// UnpackSendWarpMessageInput attempts to unpack [input] as []byte
 // assumes that [input] does not include selector (omits first 4 func signature bytes)
-func UnpackSendWarpMessageInput(input []byte) (SendWarpMessageInput, error) {
-	inputStruct := SendWarpMessageInput{}
+func UnpackSendWarpMessageInput(input []byte) ([]byte, error) {
+	inputStruct := []byte{}
 	err := WarpABI.UnpackInputIntoInterface(&inputStruct, "sendWarpMessage", input)
```

Comment:
We should be using `UnpackInput` rather than `UnpackInputIntoInterface`. See the comment: https://github.com/ava-labs/subnet-evm/blob/warp-remove-destinationChainID-destinationAddress/accounts/abi/abi.go#L216-L218

Generator should be generating something like:
```
func UnpackSendWarpMessageInput(input []byte) ([]byte, error) {
	res, err := ContractABI.UnpackInput("sendWarpMessage", input)
	if err != nil {
		return []byte{}, err
	}
	unpacked := *abi.ConvertType(res[0], new([]byte)).(*[]byte)
	return unpacked, nil
}
```